src/test/java/edu/hm/hafner/analysis/parser/xmlparser/XmlParserTest.java

```diff
@@ -0,0 +1,145 @@
+                .hasOrigin("Origin 1")
+                .hasReference("Reference 1")
+                .hasFingerprint("Fingerprint 1")
+                .hasAdditionalProperties("Property 1, Property 2");
+
+
+
+        softly.assertThat(iterator.next())
+                .hasFileName("File 2")
+                .hasLineStart(21)
+                .hasLineEnd(22)
+                .hasColumnStart(23)
+                .hasColumnEnd(24)
+                .hasCategory("Category 2")
+                .hasType("Type 2")
+                .hasSeverity(Severity.ERROR)
+                .hasMessage("Message 2")
+                .hasDescription("Description 2")
+                .hasPackageName("Package 2")
+                .hasModuleName("Module 2")
+                .hasOrigin("Origin 2")
+                .hasReference("Reference 2")
+                .hasFingerprint("Fingerprint 2")
+                .hasAdditionalProperties("Property 21, Property 22");
+
+        softly.assertThat(iterator.next())
+                .hasFileName("-")
+                .hasLineStart(0)
+                .hasLineEnd(0)
+                .hasColumnStart(0)
+                .hasColumnEnd(0)
+                .hasCategory("-")
+                .hasType("-")
+                .hasSeverity(Severity.WARNING_LOW)
+                .hasMessage("-")
+                .hasDescription("-")
+                .hasPackageName("-")
+                .hasModuleName("-")
+                .hasOrigin("-")
+                .hasReference("-")
+                .hasFingerprint("-")
+                .hasAdditionalProperties("-");
+
+    }
+
+    @Override
+    protected XmlParser createParser() {
+        return new XmlParser();
+    }
+
+    @Test
+    void shouldParseWithCustomPathAndMapper() {
+        XmlParser parser = new XmlParser(CUSTOM_PATH, new CustomXmlMapper());
+        Report report = parser.parse(createReaderFactory(ISSUES_CUSTOM_FILE));
+        Iterator<Issue> iterator = report.iterator();
+        assertSoftly(softly -> {
+            softly.assertThat(report)
+                    .hasSize(1);
+            softly.assertThat(iterator.next())
+                    .hasFileName("File 1")
+                    .hasLineStart(1)
+                    .hasLineEnd(2)
+                    .hasColumnStart(3)
+                    .hasColumnEnd(4)
+                    .hasCategory("Category 1")
+                    .hasType("Type 1")
+                    .hasSeverity(Severity.WARNING_LOW)
+                    .hasMessage("Message 1")
+                    .hasDescription("Description 1")
+                    .hasPackageName("Package 1")
+                    .hasModuleName("Module 1")
+                    .hasOrigin("Origin 1")
+                    .hasReference("Reference 1")
+                    .hasFingerprint("Fingerprint 1")
+                    .hasAdditionalProperties("Property 1, Property 2");
+        });
+
+    }
+
+    @Test
+    void shouldThrowParserException() {
```

Comment:
Can you add one test that shows that the parser still produces a list of issues if just one XML tag contains wrong values?